test/integration/oauth_basicauth_test.go

```diff
@@ -32,7 +33,12 @@ const (
 )
 
 var (
-	basicAuthCerts = map[string][]byte{
+	remotePrefix       = "remote"
```

Comment:
I don't see a good way to do it without duplication which would be hard to keep in sync. I'm removing remotePrefix though, since it was only being used in another package var anyway.